src/Orleans.sln

```diff
@@ -271,4 +271,7 @@ Global
 	GlobalSection(NestedProjects) = preSolution
 		{0BB483EF-6401-424C-9C4F-5750B300C7FD} = {DD5D35C0-A5FE-41CC-A10D-CAAA37996479}
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		EnterpriseLibraryConfigurationToolBinariesPathV6 = packages\EnterpriseLibrary.TransientFaultHandling.6.0.1304.0\lib\portable-net45+win+wp8
```

Comment:
Correct, TransientFaultHandling is probably the reason. Still, this is only useful when using the Config Tool, which we don't do. So we should remove it. There were some annoying source control side effects wrt to having things in the solution's GlobalSection (they were all related to TFVC, not Git, but still)